test/chain/tezos/lexer/samples/p005-opcodes/chain_id.hex

```diff
@@ -0,0 +1 @@
+011ac5e07d5772f15fb08bcc5a58fa1120cb4a81cc2f3411a598729d9045f81e6d004ddc055bd39eaa7f9a372586ef8980400786bb53a09c0101e852950280ade204ff0003cb7d7842406496fc07288635562bfd17e176c400000020020000001b0500036c0501036c0502020000000c037503200316053d036d0342000000290100000024747a31536a694a444842664459384a51587334414b374a6d6b3471716258384c4b77366a
```

Comment:
Cut off the first 176 chars: `011ac5e07d5772f15fb08bcc5a58fa1120cb4a81cc2f3411a598729d9045f81e6d004ddc055bd39eaa7f9a372586ef8980400786bb53a09c0101e852950280ade204ff0003cb7d7842406496fc07288635562bfd17e176c4`